tests/tokenized/BINARY_OP.txt

```diff
@@ -0,0 +1,5 @@
+def add ( a , b ) : <EOL>
+<INDENT>
+return a + b <EOL>
+<OUTDENT>
+add ( 1 , 3 ) <EOL>

```

Comment:
This file is failing the diff on Linux because it has Windows (CR+LF) line endings, which the Linux `diff` tool doesn't understand.  You can use something like dos2unix on  it, or configure your editor to save it with Linux (LF only) line endings.